### PR TITLE
[T3A2][W11-A2] Ho Yi Hang

### PR DESCRIPTION
**Pull request by yihangho**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/ui/TextUi.java [hunk 1]
```diff
@@ -123,10 +117,10 @@ public void showToUser(String... message) {
      */
     public void showResultToUser(CommandResult result) {
         final Optional<List<? extends ReadOnlyPerson>> resultPersons = result.getRelevantPersons();
-        if(resultPersons.isPresent()) {
+        if (resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedbackToUser(), DIVIDER);
     }
```
Comment (cmkumar87) [inline src/seedu/addressbook/ui/TextUi.java]: good catch!

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -88,24 +90,16 @@ public String getUserCommand() {
...
-        showToUser(
-                DIVIDER,
-                DIVIDER,
-                MESSAGE_WELCOME,
-                version,
-                MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
-                storageFileInfo,
+        showToUser(DIVIDER, DIVIDER, MESSAGE_WELCOME, version, MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE, storageFileInfo,
                 DIVIDER);
     }
```
Comment (cmkumar87) [inline src/seedu/addressbook/ui/TextUi.java]: pls check the formatting.

**Comment by cmkumar87** [comment]
@yihangho Please acknowledge and close PR. We are now advised to follow this protocol for PR closing to ensure student read our comments.

File: src/seedu/addressbook/ui/TextUi.java (hunk 2 above)
Comment (yihangho) [inline src/seedu/addressbook/ui/TextUi.java]: Oops, missed out this one. The `DIVIDER` of line 96 should match the first `DIVIDER` of line 95, am I right?

**Pull request closed without merging**